ultravox/data/datasets.py

```diff
@@ -192,7 +192,7 @@ class VoiceDatasetArgs:
     """If `prompt` is not set, the number of canned prompts to use."""
     include_audio: bool = True
     """Whether to include audio in the samples."""
-    include_context: bool = False
+    include_context: bool = True
```

Comment:
I didn't see a point in keeping this as False by default (iow I don't see many places where we want it to be False).